plugins/header-classifier/src/main/resources/META-INF/services/io.buoyant.linkerd.ResponseClassifierInitializer

```diff
@@ -0,0 +1 @@
+io.buoyant.http.classifiers.HeaderClassifierInitializer
```

Comment:
tioli: similarly it could be helpful to explain why this file is required & how linkerd uses it.